index-fr.html

```diff
@@ -196,7 +196,7 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
       </div>
 
       <div class="caption">
-        Les quatre modes de transmission de COVID-19. Cela suppose un masque efficace à 50%.
+        Les quatre modes de transmission du virus du COVID-19. Cela suppose un masque efficace à 50%.
```

Comment:
To be consistent with the rest of the document, let's just write "transmission du COVID-19".

I know that COVID-19 is the disease, and what gets transmitted is the SARS-Cov-2 virus, but at this point a lot of public discussion simply conflates the virus and the disease, and this document is no exception. And I don't think introducing the distinction will significantly improve the document.